### ISSUE TITLE
SR only label is incorrect in My Listing

### ISSUE THREAD

**Issue by cam156** [projecthydra/sufia#1275]
The screen reader on label is incorrect in the My File Listing.  Instead of saying how many files you have it indicates how many files are on the page.

I see:
`<span class="sr-only">You are currently listing your files .  You have 10 files </span>`
Even though I have more than 170 files

Se bellow image includes 17 pages (of 10) and only has 10 items in the sr tag.

<img width="765" alt="screen shot 2015-08-07 at 2 16 28 pm" src="https://cloud.githubusercontent.com/assets/1599081/9142620/f6524c06-3d0e-11e5-9c8a-48cbb0a21bf2.png">

**Comment by mjgiarlo** [comment]
Please verify if this is still the case in 7.0.0. Close if not.

**Comment by njaffer** [comment]
In Sufia 7, the file is replaced with Works (isn't it?) I cannot even access the page dashboard/files. I believe this might be the issue for Sufia 6.

**Comment by njaffer** [comment]
@mjgiarlo, I looked at Sufia 7.0.0 and I cannot reproduce this error. There is no files dashboard, I looked for sr-only label through out the code and did not find a reference that is not working. Is there anything more to it or can it be closed?

**Comment by mjgiarlo** [comment]
Let's close it. Thanks!
